### PR TITLE
Remove unused outstanding fees fields

### LINKED ISSUE #361: Update transfer related function signature
Potentially nothing needs to change, but check.
The tests will need change.

see: https://github.com/trustlines-protocol/clientlib/issues/272

### PR DESCRIPTION
**Pull request by weilbith**
Closes: #363, closes #361

**Notes:**
  - Includes cherry-picked commit by Bernd to fix the tests by hard-coding (temporally fix).
  - This requires/reacts to the most recent changes in the trustlines-protocol/contracts changes. Unfortunately does the latest `trustlines-contracts-bin` library version not include them now. There is also no such of a `dev` version like for _NodeJS_ (relevant for trustlines-protocol/clientlib). To check the changes, an up to date `contracts.json` must be injected into the virtual environment at `./venv/trustlines-contracts/build`.

### CODE REVIEW

File: tests/chain_integration/test_currency_network.py [hunk 1]
```diff
@@ -131,9 +127,6 @@ def test_get_transfer_event(currency_network_with_events, accounts, test_extra_d
     assert transfer_event.value == 10
     assert transfer_event.to == accounts[0]
     assert transfer_event.from_ == accounts[1]
-    assert transfer_event.user == accounts[0]
-    assert transfer_event.counter_party == accounts[1]
-    assert transfer_event.direction == "received"
```
Comment (cducrest) [inline tests/chain_integration/test_currency_network.py]: Why was this removed?

File: tests/gen_testdata.py [hunk 2]
```diff
@@ -158,7 +158,7 @@ def compute_one_result(
 
         for a, b, balance in zip(addresses, addresses[1:], balances_before):
             self.contract.functions.setAccount(
-                a, b, 100_000_000, 100_000_000, 0, 0, 0, 0, 0, balance
+                a, b, 100_000_000, 100_000_000, 0, 0, balance
```
Comment (cducrest) [inline tests/gen_testdata.py]: I guess this should be `a, b, 100_000_000, 100_000_000, 0, 0, False, 0, balance` idk why it was broken before or what it is used for. https://github.com/trustlines-protocol/contracts/blob/master/contracts/tests/TestCurrencyNetwork.sol#L71

File: tests/chain_integration/test_currency_network.py (hunk 1 above)
Comment (weilbith) [reply tests/chain_integration/test_currency_network.py]: Sry I trimmed it to the contract event parameters. But they haven't been touched and those are artificial parameter by the relay as I understand now. I recovered them.

File: tests/gen_testdata.py (hunk 2 above)
Comment (weilbith) [reply tests/gen_testdata.py]: :heavy_check_mark:

**Comment by weilbith** [comment]
**Status:**
This PR can't be merged at the moment, because it is based on the most recent contracts which are not supported yet by the `trustlines-contracts-bin` library.
Anyways @cducrest asked me to tidy up the history already.

**Comment by weilbith** [comment]
The approval on the actual changes has been given. The waited for update of the dependencies has been done and the tests are fixed. Thereby I will rebase this on `master` with the new `src` based directory structure.
